postgres-appliance/postgres_ha.sh

```diff
@@ -74,20 +74,6 @@ write_postgres_yaml
 
 write_archive_command_environment
 
-function noterm
-{
-	echo "Received TERM signal, but not doing anything"
-}
-
-# resurrect etcd if it's gone
-(
-  trap noterm SIGTERM
-  while true
-  do
-    etcd -name "proxy-$SCOPE" -proxy on  --data-dir=etcd -discovery-srv $ETCD_DISCOVERY_URL
```

Comment:
What would be the downside of leaving etcd proxy running? With it, we can always get to the container and find out the data in etcd with etcdctl without going through A-records in DNS.